consensus/reactor.go

```diff
@@ -581,9 +579,11 @@ func (conR *ConsensusReactor) gossipDataForCatchup(logger log.Logger, rs *cstype
 			Round:  prs.Round,  // Not our height, so it doesn't matter.
 			Part:   part,
 		}
-		logger.Debug("Sending block part for catchup", "round", prs.Round)
+		logger.Debug("Sending block part for catchup", "height", prs.Height, "round", prs.Round, "index", index)
```

Comment:
`prs.Height` already included. it is coming from `heightLogger := logger.With("height", prs.Height)`